instrumentation/graphql-java/graphql-java-20.0/library/src/main/java/io/opentelemetry/instrumentation/graphql/v20_0/GraphQLTelemetryBuilder.java

```diff
@@ -27,11 +31,31 @@ public GraphQLTelemetryBuilder setSanitizeQuery(boolean sanitizeQuery) {
     return this;
   }
 
+  /** Sets whether spans are created for GraphQL Data Fetchers. Default is {@code false}. */
+  @CanIgnoreReturnValue
+  public GraphQLTelemetryBuilder createSpansForDataFetchers(boolean createSpansForDataFetchers) {
+    this.createSpansForDataFetchers = createSpansForDataFetchers;
+    return this;
+  }
+
+  /** Sets whether spans are created for Trivial GraphQL Data Fetchers. Default is {@code false}. */
```

Comment:
it might help to explain what a trivial data fetcher is a or link to graphql documentation where it is explained.